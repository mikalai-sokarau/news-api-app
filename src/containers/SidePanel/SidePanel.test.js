import React from 'react';
import { SidePanel } from './index';
import { shallow } from 'enzyme';

describe('<SidePanel />', () => {
  test('should be rendered', () => {
    expect(shallow(<SidePanel getNewsFrom={() => {}} />)).toMatchSnapshot();
  });
});
