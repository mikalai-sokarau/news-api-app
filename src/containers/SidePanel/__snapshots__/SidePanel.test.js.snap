// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SidePanel /> should be rendered 1`] = `
<div
  className="col s4"
>
  <NewsSources />
  <SideNews
    getNewsFrom={[Function]}
  />
</div>
`;
